src/main/target/OMNIBUSF7/target.h

```diff
@@ -20,6 +20,9 @@
 
 #pragma once
 
+#define USE_MCO
+#define MCO2_PIN PC9
```

Comment:
We should really avoid adding more defines that we will just have to rip out again before we release 4.0. Just hardcode this to `NONE`, and then dynamically configure them at runtime.